.travis.yml

```diff
@@ -0,0 +1,13 @@
+dist: xenial
```

Comment:
Is it not worth actually building the app in Travis as we do with runner?. Without pyenv we wouldn't be able to exercise the build on 3.7 either.
```
- pipenv install --dev --deploy
- pipenv check
```